packages/xlsx-renderer-cli/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "sxr",
+  "version": "0.0.1-1",
+  "license": "MIT",
+  "description": "Command line tool for generating XLSX files from a template. Part of XToolset",
+  "main": "lib/program.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Siemienik/XToolset",
+    "directory": "packages/xlsx-renderer-cli"
+  },
+  "keywords": [
+    "xlsx",
+    "cli",
+    "excel",
+    "template",
+    "render",
+    "generator",
+    "view model",
+    "spreadsheet",
+    "sheet",
+    "view engine",
+    "xlsx view engine"
+  ],
+  "author": "Paweł Siemienik <siemienik.pawel@gmail.com>",
+  "scripts": {
+    "build": "rm -rf lib/* && tsc",
+    "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\"",
+    "lint": "tslint -p tsconfig.json",
+    "lint:fix": "tslint --fix -p tsconfig.json",
+    "prepublishOnly": "npm run lint && npm run build",
+    "preversion": "npm run lint",
+    "version": "npm run format && git add -A src",
+    "postversion": "git push && git push --tags"
```

Comment:
Why git push duplicated?

Reply:
both operations are separated to each themselves.

Complex description is available https://stackoverflow.com/a/3745250/4747028

I use `git push && git push --tags` instead of more modern ways because, that way always works (independently of git version).